lib/Rides.dart

```diff
@@ -109,63 +109,63 @@ class RidesStateless extends StatelessWidget {
-                              padding: const EdgeInsets.only(
-                                  top: 32, left: 16, right: 16),
-                              child: Text(
-                                  DateFormat('yMMMM').format(DateTime.now()),
-                                  style: Theme.of(context).textTheme.headline5),
-                            ),
-                            SizedBox(height: 32),
-                            currentRides.length > 0
-                                ? ridesInProgress(context)
-                                : Container(),
-                            selectedRides.isEmpty
-                                ? Padding(
-                                    padding: EdgeInsets.only(bottom: 32),
-                                    child: rideCards(context, remainingRides),
-                                  )
-                                : Container()
-                          ]),
-                    ),
-                  ),
-                  selectedRides.isNotEmpty
-                      ? Positioned(
-                          bottom: 32,
-                          child: SizedBox(
-                            width: MediaQuery.of(context).size.width,
-                            child: Padding(
-                              padding:
-                                  const EdgeInsets.only(left: 34, right: 34),
-                              child: FlatButton(
-                                  padding: EdgeInsets.all(16),
-                                  color: Colors.black,
-                                  child: Text(
-                                      'Drop off ' +
-                                          (selectedRides.length == 1
-                                              ? selectedRides[0].rider.firstName
-                                              : 'Multiple Passengers'),
-                                      style: TextStyle(
-                                          color: Colors.white,
-                                          fontSize: 16,
-                                          fontWeight: FontWeight.bold)),
-                                  onPressed: onDropoff),
-                            ),
-                          ),
-                        )
-                      : Container()
-                ],
-              ));
+          children: [
+            Container(
+              height: MediaQuery.of(context).size.height,
+              child: SingleChildScrollView(
+                child: Column(
+                    crossAxisAlignment: CrossAxisAlignment.start,
+                    children: [
+                      Padding(
+                        padding: const EdgeInsets.only(
+                            top: 32, left: 16, right: 16),
+                        child: Text(
+                            DateFormat('yMMMM').format(DateTime.now()),
+                            style: Theme.of(context).textTheme.headline4),
```

Comment:
Same issue here with using headline4 vs the carriage theme

Reply:
Fixed